cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Actions.hs

```diff
@@ -27,25 +27,36 @@ import           Control.Monad.IO.Class (liftIO)
 import           Control.Monad.Trans.Except.Extra
 import           Data.Function
 import qualified Data.Map.Strict as Map
+import Data.Word
 
 runGovernanceActionCmds :: ()
   => GovernanceActionCmds era
   -> ExceptT GovernanceActionsError IO ()
 runGovernanceActionCmds = \case
-  GovernanceActionCreateConstitution cOn newConstitution ->
-    runGovernanceActionCreateConstitution cOn newConstitution
+  GovernanceActionCreateConstitution
+      cOn network deposit anyStake mPrevGovActId proposalUrl
```

Comment:
regarding my previous comment, here you could use `GovernanceActionCreateConstitution{..}` and save some boilerplate